core/orchestrator.go

```diff
@@ -100,27 +100,37 @@ func (orch *orchestrator) ProcessPayment(payment net.Payment, manifestID Manifes
 		return nil
 	}
 
-	ticket := &pm.Ticket{
-		Recipient:         ethcommon.BytesToAddress(payment.Ticket.Recipient),
-		Sender:            ethcommon.BytesToAddress(payment.Ticket.Sender),
-		FaceValue:         new(big.Int).SetBytes(payment.Ticket.FaceValue),
-		WinProb:           new(big.Int).SetBytes(payment.Ticket.WinProb),
-		SenderNonce:       payment.Ticket.SenderNonce,
-		RecipientRandHash: ethcommon.BytesToHash(payment.Ticket.RecipientRandHash),
-	}
-	seed := new(big.Int).SetBytes(payment.Seed)
+	for i, t := range payment.Tickets {
+		ticket := & pm.Ticket{
+			Recipient: 			ethcommon.BytesToAddress(t.Recipient),
+			Sender:				ethcommon.BytesToAddress(t.Sender),
+			FaceValue: 			new(big.Int).SetBytes(t.FaceValue),
+			WinProb:			new(big.Int).SetBytes(t.WinProb),
+			SenderNonce:		t.SenderNonce,
```

Comment:
> There seems to be a styling issue in orch.ProcessPayment according to github, but not for me locally. Does the repo include a linter

We do use a linter [as a part of CI](https://github.com/livepeer/go-livepeer/blob/master/.circleci/config.yml#L37), but at the moment it is only enabled for the `pm` package because we only recently added the linter and there are a bunch of areas of the codebase that do not adhere to the standard linter rules. So, we only enabled the linter for new packages with the plan to gradually update the codebase to adhere to the standard linter rules.

Regarding the style issue here: do you have the Go plugin for VSCode installed? If you do, normally the plugin should auto format the file according to standard style rules upon saving.